Add email with important details about the competition (#108)
srobo/team-emails: SR2022/2022-04-02-competition-details-reminder.md

Please keep an eye on this as we get closer to the competition, as the content may change.

## Robot design considerations

At the competition, all robots undergo a safety check. This not only confirms that the robot satisfies our safety regulations, but also the rules. Of note are a particular few we often see teams forgetting until the last minute:

- Make sure there are 2 spare USB ports: 1 for your code, and 1 for our "Competition USB". This Competition USB tells your robot that it's in the competition and which corner it's starting in (and thus populates the `R.zone` property). You will not be able to compete without this.
- Make sure you have a way to mount the flag. We will have a few spare mounts, but please do not rely on this.
- For safety, please make sure the power button is obvious and easily accessible. In the event of an issue in the arena, we don't want to be hunting for the switch as we try and quickly turn off your robot.
- Ensure your battery is protected, ideally housed in its own rigid container

> - Make sure your robot fits inside the size constraints. If it doesn't, it will have a meeting with the _saw of shame_.

Reviewer: It is worth thinking about the order here. The first thing should be the most important.

My feeling is they should be ordered according to:
1. safety
2. rules compliance
3. functional

A robot without an accessible power button or an exposed battery is a danger everywhere, be it the arena, the test arena, or the pits.
A robot that's too large will disqualify teams from matches and be inconvenient.
A robot without enough USB ports is ultimately safe, and is rules compliant, just not very useful.

[Download our Media Consent form](https://studentrobotics.org/resources/sr2022/media-consent.pdf)

> Please bring this form with you to the competition. We will have spares at reception, but bringing completed forms will allow you, your competitors and guests to proceed through faster. Depending on whether you give consent or not, we will issue you with a wristband or lanyard. These will ensure those not providing consent are not in any of our media.

Reviewer: "these forms"

> Please bring this form with you to the competition. We will have spares at reception, but bringing completed forms will allow you, your competitors and guests to proceed through faster. Depending on whether you give consent or not, we will issue you with a wristband or lanyard. These will ensure those not providing consent are not in any of our media.

Reviewer: Don't highlight that that can not give consent.

On Sunday 24th April, Southampton is holding a marathon event, the route of which goes right through the university.

> When arriving on Sunday morning, please be prompt and arrive as close as possible to the doors opening. This not only ensures there will be parking, but that the campus won't be crowded with the general public. Additionally, it's possible you will only be able to approach the university from the west, due to the route closing part of University. The schedule is available on [our website](https://studentrobotics.org/events/sr2022/competition/).

Reviewer: This contains a mix of things. The schedule really wants to be separate and probably much earlier in the e-mail

> When arriving on Sunday morning, please be prompt and arrive as close as possible to the doors opening. This not only ensures there will be parking, but that the campus won't be crowded with the general public. Additionally, it's possible you will only be able to approach the university from the west, due to the route closing part of University. The schedule is available on [our website](https://studentrobotics.org/events/sr2022/competition/).

Reviewer: This whole marathon section feels inaccurate, especially as the details are still being handled.

We can't guarantee what the state of the parking will be or how much public will be around, or how many marathon organisers. To be honest, I think it could be quite busy regardless.
It is guaranteed they will only be able to approach from the west.
Nit pick: "Southampton" is not holding a marathon event, though I grant you it is hosting one.

I would suggest just saying:
"""
On Sunday 24th April a marathon event is being held in Southampton. As a result there will be a number of road closures in the area, including on the university campus. Parking will also be limited. Keep an eye on the competition page on our website to see up-to-date information on how to reach the competition.

Access to food sellers may also be restricted, so please plan accordingly.
"""

> Due to this road closure, it will be difficult to get to the food. We strongly recommend bringing a packed lunch with you on Sunday.

Reviewer: > it will be difficult to get to the food
Word missing

---
to: Student Robotics 2022
subject: Competition details and important reminders
---

Hello all!

It's only 2 weeks until the big day - [the competition](https://studentrobotics.org/events/sr2022/competition/)!

Before then, we have a few important updates and reminders for you:

## Robot design considerations

At the competition, all robots undergo a safety check. This not only confirms that the robot satisfies our safety regulations, but also the rules. Of note are a particular few we often see teams forgetting until the last minute:

- Please make sure the power button is obvious and easily accessible. In the event of an issue in the arena, we don't want to be hunting for the switch as we try and quickly turn off your robot.

> - Ensure your battery is protected, ideally housed in its own rigid container

Reviewer 2: ```suggestion
- Ensure your battery is protected, ideally housed in its own rigid container.
```

## Southampton Marathon

On Sunday 24th April, a marathon event is being held in Southampton. As a result there will be a number of road closures in the area, including on the university campus. Parking will also be limited. Keep an eye on the [competition page](https://studentrobotics.org/events/sr2022/competition/) on our website to see up-to-date information on how to reach the competition.

> Access to food sellers may also be restricted, so please plan accordingly.

Reviewer 3: ```suggestion
Please note that this also means that access to food sellers may also be restricted, so please plan accordingly.
```

## Robot design considerations

At the competition, all robots undergo a safety check. This not only confirms that the robot satisfies our safety regulations, but also the rules. Of note are a particular few we often see teams forgetting until the last minute:

> - For safety, please make sure the power button is obvious and easily accessible. In the event of an issue in the arena, we don't want to be hunting for the switch as we try and quickly turn off your robot.

Reviewer: Remove "For safety"

> As with previous years, we require you to sign a media consent form. These media consent forms allow us to use videos and photographs taken at the event for future marketing purposes. However, the process has changed slightly.

Reviewer: Remove "As per previous years"

> As with previous years, we require you to sign a media consent form. These media consent forms allow us to use videos and photographs taken at the event for future marketing purposes. However, the process has changed slightly.

Reviewer: We need everyone to sign a media consent form

- Ensure your battery is protected, ideally housed in its own rigid container
- Make sure you have a way to mount the flag. We will have a few spare mounts, but please do not rely on this.
- Make sure your robot fits inside the size constraints. If it doesn't, it will have a meeting with the _saw of shame_.
- Make sure there are 2 spare USB ports: 1 for your code, and 1 for our "Competition USB". This Competition USB tells your robot that it's in the competition and which corner it's starting in (and thus populates the `R.zone` property). You will not be able to compete without this.

All of these and more can be found in our [safety regulations](https://studentrobotics.org/docs/kit/safety-regulations) and [rules](https://studentrobotics.org/rules).

## Media consent

As with previous years, we require you to sign a media consent form. These media consent forms allow us to use videos and photographs taken at the event for future marketing purposes. However, the process has changed slightly.

[Download our Media Consent form](https://studentrobotics.org/resources/sr2022/media-consent.pdf)

> Please bring this form with you to the competition. We will have spares at reception, but this will allow you, your competitors and guests to proceed through faster. Depending on whether you give consent or not, we will issue you with a wristband or lanyard, so we ensure you are not in any of our media.

Reviewer 3: ```suggestion
Please bring this form with you to the competition. We will have spares at reception, but bringing a completed form will allow you, your competitors and guests to proceed through faster. Depending on whether you give consent or not, we will issue you with a wristband or lanyard. These will ensure those not providing consent are not in any of our media.
```

Reviewer: This kind implies that 1 form might cover the whole team plus guests, which is inaccurate
